core/README.md

```diff
@@ -81,10 +83,10 @@ timestamp of the actual execution.
 > [timestamps extension](https://github.com/fiboa/timestamps).
 
 **determination_method**: Allowed values:
-- `manual` (hand drawn from imagery)
-- `driven`
-- `surveyed`
-- `administrative`
-- `auto-operation` (auto-created from operation)
-- `auto-imagery` (auto-created from imagery)
-- `unknown`
+
+- `manual`: Hand created from imagery, e.g. using a tool to point and click on a map.
+- `surveyed`: Determined through a professional land survey measuring the actual distances and angles on the ground.
+- `driven`: An operator physically drove (or walked) around the perimeter of the field with a geopositioning device (e.g. GPS) that records the path taken. The operator actively and explicitly selected in the terminal to create a boundary.
```

Comment:
Comment from @hannah-rae:

> Instead of “driven” maybe “traversed”? It would include driving and walking that way

Reply:
Walking is included in the description, but for the sake of aligning with ADAPT/AgGateway we should probably keep the name of the value itself.